client/src/pages/NewComment/index.tsx

```diff
@@ -41,7 +44,33 @@ const NewComment: React.FunctionComponent<NewCommentProps> = ({
     };
 
     if (courseId && postId) {
-      await addComment(courseId, postId, commentBody);
+      if (comment?.id) {
+        await axios.put(
+          `/comments/${courseId}/posts/${postId}/comments/${comment?.id}`,
```

Comment:
the params are serialized from the params arguments anyway so you don't need /${courseId}/posts/${postId}/comments/${comment?.id}